client/app/bundles/course/survey/pages/ResponseShow/index.jsx

```diff
@@ -119,8 +127,35 @@ class ResponseShow extends React.Component {
     );
   }
 
+  renderSubmissionInfo() {
+    const { response } = this.props;
+    return (
+      <Table style={styles.submissionInfoTable}>
+        <TableBody displayRowCheckbox={false} stripedRows>
+          <TableRow selectable={false}>
+            <TableHeaderColumn>Student</TableHeaderColumn>
+            <TableRowColumn>{response.name}</TableRowColumn>
+          </TableRow>
+          <TableRow selectable={false}>
+            <TableHeaderColumn>Submitted At</TableHeaderColumn>
+            <TableRowColumn>
+              <FormattedDate
```

Comment:
Could you use `moment` instead? We want to move away from `FormattedDate` because it's clunky to use.